samples/dashboards/demo/crossfilter/demo.js

```diff
@@ -0,0 +1,226 @@
+                options: {
+                    csv: document.getElementById('csv').innerHTML,
+                    decimalPoint: '.',
+                    itemDelimiter: ',',
+                    dataModifier: {
+                        type: 'Range',
+                        ranges: [{
+                            column: 'Country',
+                            minValue: 'Afghanistan',
+                            maxValue: 'Zimbabwe'
+                        }, {
+                            column: 'Year',
+                            minValue: 1995,
+                            maxValue: 2020
+                        }]
+                    }
+                }
+            }]
+        },
+        gui: {
+            layouts: [{
+                rows: [{
+                    cells: [{
+                        id: 'Top'
+                    }]
+                }, {
+                    cells: [{
+                        id: 'Middle-left'
+                    }, {
+                        id: 'Middle-right'
+                    }]
+                }, {
+                    cells: [{
+                        id: 'Bottom'
+                    }]
+                }]
+            }]
+        },
+        components: [{
+            cell: 'Top',
+            type: 'HTML',
+            elements: [
+                '<h2 style="text-align:center">Economic Acitivity</h2>'
+            ]
+        }, {
+            cell: 'Middle-left',
+            type: 'Highcharts',
+            connector: {
+                id: 'Countries'
+            },
+            columnAssignment: {
+                Country: 'x',
+                Agriculture: 'y',
+                Industry: 'y',
+                Services: 'y'
+            },
+            chartOptions: {
+                title: {
+                    text: 'Countries'
+                },
+                subtitle: {
+                    text: 'Select a range'
+                },
+                chart: {
+                    zooming: {
+                        type: 'x'
+                    }
+                },
+                plotOptions: {
+                    series: {
+                        marker: {
+                            enabled: false,
+                            radius: 1
+                        },
+                        stickyTracking: false
+                    }
+                },
+                legend: {
+                    align: 'right',
+                    verticalAlign: 'top'
+                },
+                tooltip: {
+                    positioner: () => ({ x: 10, y: 10 }),
+                    shadow: false,
+                    shared: true,
+                    valueSuffix: '%'
+                },
+                yAxis: {
+                    visible: false
+                },
+                xAxis: {
+                    type: 'category',
+                    minRange: 0.8,
+                    labels: {
+                        rotation: 90
```

Comment:
Is it just me or is it easier to tilt my head to the left instead of right? 😅

Reply:
For me it is easier as well, but then it becomes harder to read.